python/docs/librmm_api.rst

```diff
@@ -0,0 +1,4 @@
+API Reference
+==============
```

Comment:
Make a similar change to `Python API Reference` or simply `Python API` in the other rst file. Choose one of:
```suggestion
C++ API Reference
=================
```

```suggestion
C++ API
=======
```

Reply:
Given the nesting that we've discussed offline I may not make this change. Let's see how you and other reviewers feel about the nesting first then we can retitle appropriately.